### ISSUE TITLE
0.5.0 and 0.5.1 docker images don't launch

### ISSUE THREAD

**Issue by Arnavion** [Brainicism/bgutil-ytdlp-pot-provider#31]
```sh
$ docker run -it --rm --name bgutil-provider brainicism/bgutil-ytdlp-pot-provider:0.5.1

node:internal/modules/cjs/loader:1228
  throw err;
  ^

Error: Cannot find module '@commander-js/extra-typings'
Require stack:
- /app/build/main.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1225:15)
    at Module._load (node:internal/modules/cjs/loader:1051:27)
    at Module.require (node:internal/modules/cjs/loader:1311:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (/app/build/main.js:8:25)
    at Module._compile (node:internal/modules/cjs/loader:1469:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1548:10)
    at Module.load (node:internal/modules/cjs/loader:1288:32)
    at Module._load (node:internal/modules/cjs/loader:1104:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/app/build/main.js' ]
}

Node.js v20.17.0
```

Same with `:0.5.0`. `:0.4.0` works fine.

**Comment by grqz** [comment]
should be fixed in the latest release

**Comment by Brainicism** [comment]
Should be fixed in 0.5.2, thanks for the report!

**Comment by Arnavion** [comment]
Yes, I see it: https://github.com/Brainicism/bgutil-ytdlp-pot-provider/commit/be847ed37995c0b4868421a4d4d3a3b1e16854f4#diff-da00458cdaeaea2314cb0e0101c85130593048072ada62de01727958c5d6ca37R21

Confirmed that `:0.5.2` works. Thanks.
